app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Should add __init__ to initialize the message of exception and you can use Exception class as a parent, not a BaseException

Example:
class NotWearingMaskError(Exception):
    def __init__(self, message):
        super.__init__(message)
        self.message = message